packages/jsii-rosetta/lib/tablets/tablets.ts

```diff
@@ -35,16 +36,22 @@ export class LanguageTablet {
   public static async fromOptionalFile(filename: string) {
     const ret = new LanguageTablet();
     if (fs.existsSync(filename)) {
-      await ret.load(filename);
+      try {
+        await ret.load(filename);
+      } catch (e) {
+        logging.warn(`${filename}: ${e}`);
+      }
     }
     return ret;
   }
 
   private readonly snippets: Record<string, TranslatedSnippet> = {};
 
-  public addSnippet(snippet: TranslatedSnippet) {
-    const existingSnippet = this.snippets[snippet.key];
-    this.snippets[snippet.key] = existingSnippet ? existingSnippet.mergeTranslations(snippet) : snippet;
+  public addSnippet(...snippets: TranslatedSnippet[]) {
```

Comment:
`addSnippets`?

Reply:
Yeh, but that breaks API. This seemed simpler and just as effective.